src/battle/1A5830.c

```diff
@@ -2702,6 +2700,15 @@ API_CALLABLE(RemoveActor) {
 }
 
 API_CALLABLE(DropStarPoints) {
+    /// Star Point multiplier, indexed by actor count.
+    /// +10% multiplier for two actors
```

Comment:
Isn't this 10% for 3, 30% for 4+?

Reply:
Don't know how I managed to mess this one up, lol